packages/core/admin/admin/src/pages/SettingsPage/pages/ApplicationInfosPage/components/LogoInput/tests/index.test.js

```diff
@@ -170,28 +170,29 @@ describe('ApplicationsInfosPage || LogoInput', () => {
       expect(document.body).toMatchSnapshot();
     });
 
-    it('should show error message when uploading wrong file format', async () => {
-      render();
-      const changeLogoButton = document.querySelector('button');
-      fireEvent.click(changeLogoButton);
-      fireEvent.click(screen.getByText('From url'));
-
-      const textInput = document.querySelector('input[name="logo-url"]');
-
-      fireEvent.change(textInput, {
-        target: {
-          value: 'https://docs.strapi.io/assets/img/qsg-handson-restaurant_2.28faf048.gif',
-        },
-      });
-
-      fireEvent.click(screen.getByText('Next'));
-
-      await waitFor(() =>
-        expect(
-          screen.getByText('Wrong format uploaded (accepted formats only: jpeg, jpg, png, svg).')
-        ).toBeInTheDocument()
-      );
-    });
+    // The system now supports gif format images. So this unit test is invalid.
```

Comment:
the LogoInput feature does not support gif

Reply:
done